encrypted-media-respec.html

```diff
@@ -839,37 +839,42 @@
                     <p class="note">For example, if <var>audio/video type</var> is Video and the top-level type is not "video" or <var>media types</var> contains non-video codecs.</p>
                   </li>
                   <li><p>If <var>robustness</var> is not the empty string and contains an unrecognized value or a value not supported by <var>implementation</var>, continue to the next iteration. String comparison is case-sensitive.</p></li>
-                  <li><p>If the user agent and <var>implementation</var> do not support playback of encrypted <a def-id="media-data"></a> for the combination of <var>container</var>, <var>media types</var>, <var>robustness</var> and <var>local accumulated configuration</var>, continue to the next iteration.</p>
+                  <li><p>If the user agent and <var>implementation</var> definately support playback of encrypted <a def-id="media-data"></a> for the combination of <var>container</var>, <var>media types</var>, <var>robustness</var> and <var>local accumulated configuration</var>:</p>
                     <p class="note"><var>requested media capability</var> (content type and robustness) must be supported with all previously added requested media capabilities.</p>
-                  </li>
-                  <li>
-                    <p>
-                      Add <var>requested media capability</var> to <var>supported media capabilities</var>.
-                    </p>
-                    <p class="note">
-                      This step ensures that the values of the members of entries in <var>supported media capabilities</var> are exactly the strings supplied in
-                      <var>requested media capability</var> without modification by the User Agent.
-                    </p>
-                  </li>
-                  <li>
-                    <dl class="switch">
-                      <dt>If <var>audio/video type</var> is Video:</dt>
-                      <dd>
+                    <p class="note">If <var>media types</var> is empty (i.e. only container is specified) then
```

Comment:
Line 835 says _parameters_ may be empty. Rather than just allowing that and trying to handle it in this Note, perhaps we should handle it there.

We have two options for handling empty.
1. Set _parameters_ to _all_ possible parameters for _container_.
2. If _container_ implies a set of codecs and other parameters, let _parameters_ be that set. Otherwise, continue to the next iteration.

I suggest that we choose (2). See the comment at line 846 for why.